.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on: push
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup npm cache
+        uses: actions/setup-node@v3
+        with:
+          cache: "npm"
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: npm ci --audit false --fund false
```

Comment:
What does audit do and why is it turned off?

Reply:
It checks for known vulnerabilities in the dependencies it installs. I think it doesn't make sense to run that logic in this pipeline, because most likely people will only check the output of this step when it fails (audit doesn't make it fail btw), and more importantly, we will have dependabot doing the same check for us anyway in a much more visible way.

Reply:
Makes sense